radio/src/io/multi_firmware_update.h

```diff
@@ -81,9 +81,19 @@ class MultiFirmwareInformation
 
     bool isMultiExternalFirmware() const
     {
-      return ((telemetryInversion || boardType == FIRMWARE_MULTI_STM) &&
-              optibootSupport && bootloaderCheck &&
-              telemetryType == FIRMWARE_MULTI_TELEM_MULTI_TELEMETRY);
+      switch(boardType) {
+        case FIRMWARE_MULTI_STM:
+        case FIRMWARE_MULTI_AVR:
+          return (telemetryInversion && optibootSupport && bootloaderCheck &&
+                  telemetryType == FIRMWARE_MULTI_TELEM_MULTI_TELEMETRY );
+
+        case FIRMWARE_MULTI_ORX:
+          return (telemetryInversion && bootloaderCheck &&
+                  telemetryType == FIRMWARE_MULTI_TELEM_MULTI_TELEMETRY );
```

Comment:
So for all modules something like that? 
```c
return ( telemetryInversion && bootloaderCheck &&
              (boardType == FIRMWARE_MULTI_ORX || optibootSupport) &&
              telemetryType == FIRMWARE_MULTI_TELEM_MULTI_TELEMETRY );
```

Reply:
You could write it like that, I just don't find it easy to read. Can change if you find it clearer this way

Reply:
Just shorter 😬